web/models/user.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MovieWagerApi.User do
+  use MovieWagerApi.Web, :model
+
+  schema "users" do
+    field :screen_name, :string
+    field :profile_image_url, :string
+    field :twitter_id, :integer
+    field :name, :string
```

Comment:
All of these fields are provided by twitter, so I'd prefer to stick with their naming convention. I'm not sure I want to bind our id to the one provided by twitter.